config.go

```diff
@@ -150,7 +150,7 @@ type config struct {
 	RESTListeners []string `long:"restlisten" description:"Add an interface/port to listen for REST connections"`
 	Listeners     []string `long:"listen" description:"Add an interface/port to listen for peer connections"`
 	DisableListen bool     `long:"nolisten" description:"Disable listening for incoming peer connections"`
-	ExternalIPs   []string `long:"externalip" description:"Add an ip to the list of local addresses we claim to listen on to peers"`
+	ExternalIPs   []string `long:"externalip" description:"Add an ip:port to the list of local addresses we claim to listen on to peers. If a port is not specified, the default (9735) will be used regardless of other parameters"`
```

Comment:
Would it be preferable to inherit from listening port?  for example, if `--listen=:9736` is specified, and `externalip` does not specify a port, then inherit from `--listen` (use port 9736).  i'm not sure why it would be possible to set a different port for `externalip` and `listen`.

Reply:
@yancyribbens: you might listen locally on a different port than your external one using NAT or something. And if you listen on a few different ports on different interfaces then which would be default? Better to just document the current behavior IMO, its the most logical to me

Reply:
Those are good points.  Yes I see why it's better to just document.